.github/workflows/npmpublish.yml

```diff
@@ -22,3 +22,17 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{secrets.npm_token}}
           PRE_RELEASE: ${{github.event.release.prerelease}}
+  update-bug-report:
+    runs-on: ubuntu-latest
+    name: Update bug report version to ${{ github.event.release.tag_name }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+    - run: npm ci
+    - run: node scripts/updateBugReportVersion.mjs
+      env:
+        VERSION: ${{ github.event.release.tag_name }}
+    - run: git commit -m "ci(): update bug report version" -a
```

Comment:
i don't mind an automatic commit per release, till we release once in a while.
But if we release each small bugfix ( auspicable ) this becomes like the automatic changelog

Reply:
explain your concerns?
history pollution?
We can change this action to run chronologically instead, once a week or 2

Reply:
I will extract the action logic anyways to a standalone action